### PR TITLE
clean up documentation

### PR DESCRIPTION
**Pull request by botany-demo**
clean up documentation

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 README
 
-contributing guide updates
+clean up documentation
```
Comment (botany-lead) [inline readme.md:3]: much cleaner 👍

**Review by botany-lead — APPROVED** [review]
looks good to me

**Comment by botany-lead** [comment]
comment one

**Comment by botany-lead** [comment]
comment two

**Comment by botany-lead** [comment]
comment three

**Comment by botany-lead** [comment]
comment four

**Comment by botany-lead** [comment]
comment five

**Comment by botany-demo** [comment]
comment six

**Comment by botany-lead** [comment]
seven

**Pull request merged**